src/lando/main/models/repo.py

```diff
@@ -29,6 +30,11 @@
 )
 
 
+def validate_path_in_repo_root(value):
+    if not value.startswith(settings.REPO_ROOT):
+        raise ValidationError(f"Path must be in {settings.REPO_ROOT}.")
```

Comment:
Might be good to use pathlib here to do a bit more rigorous validation (e.g., check that the parent directory is the repo root.)